<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MinerGift</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="topHorizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <widget class="QLabel" name="labelQTG">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>10</y>
            <width>155</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Qu'est ce que tu GEEKes ?</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelThankYou">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>196</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Un grand merci de le part de :</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="sep1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLCDNumber" name="lcdRate">
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="digitCount">
          <number>2</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="intValue" stdset="0">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelInfoRate">
         <property name="text">
          <string>Un peu, beaucoup, passion√©ment ??</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="sliderRate">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="sliderPosition">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="sep2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bottomHorizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonStartStop">
         <property name="text">
          <string>Start / Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sliderRate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdRate</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRate</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdRate</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
